.zenodo.json

```diff
@@ -3,12 +3,42 @@
     "title": "ihmeuw/vivarium_nih_us_cvd: Archival release",
     "upload_type": "software",
     "creators": [
-//        {
-//            "orcid": "0000-0000-0000-0000",
-//            "affiliation": "Institute for Health Metrics and Evaluation",
-//            "name": "TODO: Add authors names, affiliation, and orcid (if applicable) to `creators` array"
-//        }
+        {
+            "orcid": "0000-0001-8649-7561",
+            "affiliation": "Institute for Health Metrics and Evaluation",
+            "name": "Rajan Mudambi"
+        },
+        {
+            "orcid": "0009-0004-6649-7010",
+            "affiliation": "Institute for Health Metrics and Evaluation",
+            "name": "Steve Bachmeier"
+        },
+        {
+            "orcid": "0000-0002-7556-2740",
+            "affiliation": "Institute for Health Metrics and Evaluation",
+            "name": "Hussain Jafari"
+        },
+        {
+            "orcid": "0009-0005-7858-0222",
+            "affiliation": "Institute for Health Metrics and Evaluation",
+            "name": "Sylvia Lutze"
+        },
+        {
+            "orcid": "0000-0003-1585-5100",
+            "affiliation": "Institute for Health Metrics and Evaluation",
+            "name": "Alison Bowman"
+        },
+        {
+            "orcid": "0009-0009-3789-5382",
+            "affiliation": "Institute for Health Metrics and Evaluation",
+            "name": "Caroline Kinuthia"
+        },
+        {
+            "orcid": "0000-0001-6033-4713",
+            "affiliation": "Institute for Health Metrics and Evaluation",
+            "name": "Abraham Flaxman"
+        }
```

Comment:
Matt Kappel and Kjell Swedin also contributed to this model, so they should probably be added here if we have their orcids